lib/iris/tests/test_analysis.py

```diff
@@ -365,6 +365,26 @@ def test_percentile_1d(self):
                        ('analysis', 'third_quartile_foo_1d.cml'),
                        checksum=False)
 
+    def test_fast_percentile_1d(self):
+        cube = tests.stock.simple_1d()
+
+        first_quartile = cube.collapsed('foo', iris.analysis.PERCENTILE,
+                                        percent=25, percentile_method='fast')
+        np.testing.assert_array_almost_equal(first_quartile.data,
+                                             np.array([2.5], dtype=np.float32))
+        self.assertCML(first_quartile, ('analysis',
```

Comment:
here, and in other places, I would rather not run `self.assertCML` in these cases, it is an expensive and sometimes unstable test pattern

I would also prefer to see one assert per test case.  T=in this case, that would mean having 2 test functions, one for first_quartile and one for third_quartile